apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx

```diff
@@ -3,7 +3,84 @@ import { IconCatalog } from './Components/IconCatalog/IconCatalog.tsx';
 
 <Meta title="Concepts/Migration/from v0/Icons" />
 
-# v0 - v9 Icon Catalog
+# Icons Migration
+
+## createSvgIcon
+
+V0 exports a `createSvgIcon` function that allows creating a custom icon such as:
+
+```jsx
+export const MyIcon = createSvgIcon({
+    svg: ({ classes }) => (
+        <svg role="presentation" focusable="false" viewBox="2 2 16 16" className={classes.svg}>
+            <g className={classes.outlinePart>
+                <path d="..." />
+            </g>
+            <g className={classes.filledPart}>
+                <path d="..." />
+            </g>
+        </svg>
+    ),
+    displayName: "MyIcon"
+});
+```
+
+And its usage would be like:
+
+```jsx
+// Usage Example
+
+// Default filled icon
+<MyIcon />
+
+// Outline Icon
+<MyIcon outline />
+```
+
+To achieve the same using V9 we have to make usage of a combination with `wrapIcon` and `bundleIcon`.
+
+```jsx
+// MyIcon.tsx
+import { FluentIconsProps, bundleIcon, wrapIcon } from '@fluentui/react-icons';
+
+export const MyOutlineIcon = wrapIcon((props: FluentIconsProps) => {
+  return (
+    <svg height="1rem" width="1rem" role="presentation" focusable="false" viewBox="2 2 16 16" {...props}>
+      <g>
+        <path d="..." />
+      </g>
+    </svg>
+  );
+}, 'MyOutlineIcon');
+
+export const MyFilledIcon = wrapIcon((props: FluentIconsProps) => {
+  return (
+    <svg height="1rem" width="1rem" role="presentation" focusable="false" viewBox="2 2 16 16" {...props}>
+      <g>
+        <path d="..." />
+      </g>
+    </svg>
+  );
+}, 'MyFilledIcon');
+
+export const MyIcon = bundleIcon(MyFilledIcon, MyOutlineIcon);
+```
+
+And its usage would be like:
+
+```jsx
+// Usage Example
+
+// Default outlined icon
+<MyIcon />
+
+// Filled Icon
+<MyIcon filled />
```

Comment:
let's also add some docs here about sizing and how v0 differs from v9. I think users will always have to size the v9 icons after migration right?

Reply:
Added https://github.com/microsoft/fluentui/pull/25828/commits/1e15a0c7c40150aa775e75a38c45d42b2aae1d5a